tototrinh/liferay-portal
# LPS-166514 As a Site Admistrator, I want the Sites and Portlet LAR files used for imports to be removed from the Document and Media library

Hi @tototrinh ,

Please help me review this ticket.
https://issues.liferay.com/browse/LPS-166514

Motivation
=======
There are some usecases where keeping LAR files in the Document and Media Library is not reasonable. That LAR files make the later LAR files bigger unnecessarily. We want to remove such LAR files from Document and Madia library. In this PR, it focus to solve the AC-1, AC-2, AC-3 which are mentioned in the LPS-166514


Proposed Solution
========
The AC-1 and AC-2 follow the description of AC-1 and AC-2.

With the AC-3, the LAR file in the background task will be removed immediately in the Executor classes.


Steps to Verify
========
1. Create a blank site.
1. Go to Content and Data -> Web contents -> Add a folder and 2 web contents.
1. Go to export -> export the LAR files -> download the Lar file. Check the DLFileEntry Table.
1. Go to Import -> + -> import the downloaded lar file -> Continue
1. Check in the new Import process page have two warning messages with corresponding displays:  "The import LAR file is not kept and thrown away immediately after the process." and "Import process will be deleted after 30 days."
1. Click Import -> Check the DB, it does not have any new dlFileEntry

Screenshots
=======
For the import layout :
![image](https://github.com/tototrinh/liferay-portal/assets/67891751/cd3391d0-cfdf-4d19-a9bf-0b7858c1a6fa)
For the import portlet:
![image](https://github.com/tototrinh/liferay-portal/assets/67891751/24329397-8640-4876-b0ca-0c9145ac5bde)
